src/third_parties/daveKoeller/AlphanumComparator.java

```diff
@@ -23,11 +23,12 @@
  */
 
 package third_parties.daveKoeller;
-import java.text.Collator;
-import java.util.Comparator;
 
 import com.owncloud.android.datamodel.OCFile;
 
+import java.text.Collator;

```

Comment:
Please, disable automatic formatting. Changes are mainly in two files, but the PR contains over 50 files changed due to imports in different lines.